zkevm-circuits/src/state_circuit.rs

```diff
@@ -48,9 +50,11 @@ pub struct StateConfig {
     initial_value: Column<Advice>, /* Assigned value at the start of the block. For Rw::Account
                                     * and Rw::AccountStorage rows this is the committed value in
                                     * the MPT, for others, it is 0. */
+    state_root: Column<Advice>,
     lexicographic_ordering: LexicographicOrderingConfig,
     lookups: LookupsConfig,
-    power_of_randomness: [Column<Instance>; N_BYTES_WORD - 1],
+    powers_of_lookup_randomness: [Column<Instance>; 6],
+    powers_of_word_randomness: [Column<Instance>; N_BYTES_WORD - 1],
```

Comment:
I have a bit worrying about this huge instance block. Each of these cols has to be assigned with N_ROWS values, would this bring some difficulty for the verifier, especially when we are using the aggregation circuits?